src/main/java/org/primeframework/mvc/security/guice/SecurityModule.java

```diff
@@ -64,7 +66,8 @@ protected void configure() {
     SecuritySchemeFactory.addSecurityScheme(binder(), "authorize-method", AuthorizeMethodScheme.class);
 
     // Bind the Cipher/Encryption interfaces
-    bind(CipherProvider.class).to(DefaultCipherProvider.class).asEagerSingleton();
+    bind(CipherProvider.class).annotatedWith(Names.named("CBC")).to(CBCCipherProvider.class).asEagerSingleton();
+    bind(CipherProvider.class).annotatedWith(Names.named("GCM")).to(GCMCipherProvider.class).asEagerSingleton();
```

Comment:
In theory, a user of this library could be injecting `CipherProvider`. 

To be backwards compatible, the CBC impl would need to be bound un-named. Or, at least, also bound un-named.